plugins/copy-column/src/copy-column.ts

```diff
@@ -0,0 +1,175 @@
+import api, { Flatfile } from "@flatfile/api"
+import type { FlatfileEvent, FlatfileListener } from '@flatfile/listener'
+
+export type Primitive = string | number | null | boolean
+export type SimpleRecord = Record<string, Primitive>
+
+export function copyColumn() {
+    return (listener: FlatfileListener) => {
+      listener.on('workbook:created', async ({ context: { fileId, workbookId } }: FlatfileEvent) => {
+        const { data: sheets } = await api.sheets.list({ workbookId })
+
+        sheets.forEach(async (sheet) => {
+          const loadAllFields = await getAllFields(sheets, sheet.slug);
+          setSheetActions(sheet, [
+              copyColumnValuesBlueprint(loadAllFields),
```

Comment:
Neat to populate all the field enums for each sheet! 
Does this `workbook:created` event get triggered for File based Workbooks? Would we want to add the Action on every `workbook:created` or `sheet:created`?

Reply:
I'm not seeing `sheet:created` when creating a space. What do you mean by File based Workbooks?